### ISSUE TITLE
apiParams String[] escapes strings with '\'

### ISSUE THREAD

**Issue by praveenaj** [apidoc/apidoc#379]
Hi,

`apiParams String[]` escapes all the string elements passed with a backslash `\`. This causes the API function to not work as intended. 

Expected:
``['foo','bar']``

Backend receives:
```[\'foo\', \'bar\']```

**Comment by alqu** [comment]
True. Same here.

The sample response in v0.13.x looked like that:
```json
{
    "data": [
        {
            "id": 34,
            "centerId": 100, ...
```
And since v0.14.x it looks like:
```json
"{\"data\":[{\"id\":34,\"centerId\":100, ...
```

**Comment by alqu** [comment]
OK, will be solved with #384
